communication/src/PlainUDPCommunication.cpp

```diff
@@ -164,6 +164,11 @@ class PlainUDPCommunication::PlainUdpImpl {
     // Initialize socket.
     udpSockFd = socket(AF_INET, SOCK_DGRAM, 0);
 
+    if (udpSockFd < 0) {
+      LOG_DEBUG(_logger, "Couldn't open the socket");
+      return -1;
```

Comment:
In the [code](https://github.com/vmware/concord-bft/blob/master/communication/src/PlainUDPCommunication.cpp#L178) below we terminate the application in the event of failure.
Probably, we might want to do the same here:
`ConcordAssert(udpSockFd >= 0, std::srting{"Failed to initialize socket, error: "} + strerror(errno));`